Gemfile

```diff
@@ -5,6 +5,8 @@ gem "activesupport", "< 4.0.0", group: :compat_testing, platform: "ruby"
 
 gem "chef-config", path: "chef-config" if File.exist?(__FILE__ + "../chef-config")
 
+gem "rake"
```

Comment:
Probably should have a comment to explain that we need this so rake is still available after bundling without the development group.